src/NovelRT/CMakeLists.txt

```diff
@@ -156,6 +156,19 @@ target_link_libraries(Engine
     Vulkan::Vulkan
 )
 
+install(
+  TARGETS Engine
+  EXPORT NovelRT
+  DESTINATION lib
+  RUNTIME DESTINATION bin
+)
+#Workaround due to hierarchy destruction when installing by TARGETS
```

Comment:
Post MVP I want to restructure the engine such that each "area" is its own static/object library, and is then linked into the main "engine" - this should simplify the layout of our include files and flatten them so that we can use PUBLIC_HEADER or whatever. Hopefully this'll enforce a lot more separation between various components.